### PR TITLE
Add archeo (as Blockbase child)

### PR DESCRIPTION
**Pull request by scruffian**
#### Changes proposed in this Pull Request:
This is a skeleton version of Archeo:
localhost_4759_

### CODE REVIEW

File: blockbase/theme.json [hunk 1]
```diff
@@ -535,7 +535,7 @@
 					"fontFamily": "var(--wp--preset--font-family--heading-font)",
 					"fontWeight": "var(--wp--custom--heading--typography--font-weight)",
 					"lineHeight": "var(--wp--custom--heading--typography--line-height)",
-					"fontSize": "32px"
+					"fontSize": "var(--wp--preset--font-size--huge)"
```
Comment (MaggieCabrera) [inline blockbase/theme.json]: is this coming from elsewhere or just a fix?

Comment (scruffian) [reply blockbase/theme.json]: It's just a fix. I'll create a separate PR for it.

Comment (scruffian) [reply blockbase/theme.json]: https://github.com/Automattic/themes/pull/5101

**Comment by scruffian** [comment]
@kjellr @beafialho would you mind taking a look at this and seeing that else needs to be done?

From the top of my head there's:
- Patterns
- Color palettes

File: archeo/style.css [hunk 2]
```diff
@@ -0,0 +1,16 @@
+/*
+Theme Name: Archeo
+Theme URI: https://github.com/Automattic/themes/tree/trunk/archeo
+Author: Automattic
+Author URI: https://automattic.com
+Description: A clean portfolio theme
```
Comment (scruffian) [inline archeo/style.css:6]: We will probably need to update this.

**Comment by beafialho** [comment]
> @kjellr @beafialho would you mind taking a look at this and seeing that else needs to be done?

Besides Patterns and Color palettes, have you added the typography styles? I also don't see any Post mockups on Figma (Small Theme V3), do we need those?

**Comment by scruffian** [comment]
This does include the typography styles.

We don't have post mockups in the Figma, but it might be easier/quicker to do them directly in this PR. What are the changes needed?

**Comment by beafialho** [comment]
> We don't have post mockups in the Figma, but it might be easier/quicker to do them directly in this PR.

How's that? I think we may have to iterate a bit on those.

**Comment by scruffian** [comment]
> How's that? I think we may have to iterate a bit on those.

Whatever is easier for you :)

**Comment by beafialho** [comment]
@scruffian [here's](https://www.figma.com/file/28iXII44cLy2oKwyHLgTdi/Archeo?node-id=1841%3A1452) the post page for Archeo.

Single Post

**Comment by kjellr** [comment]
@beafialho can you include a mockup of when there's no featured image just so that's super clear? I figure the right half of the screen would just be dark gray.

**Comment by beafialho** [comment]
Yes, here's a mockup of when there's no featured image.

Single Post (without Featured Image)

**Comment by mikachan** [comment]
Closing this in favor of #5436.

I've copied the single post mockups into a separate issue: #5423.

**Pull request closed without merging**